coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -92,6 +92,11 @@
         'args': '--new-window',
         'gui': True
     },
+    'code': {
```

Comment:
Are you sure about `'code'`?

Reply:
@sangamcse do i require to close this pr and then file a pr again from a new branch?

Reply:
No. Never close any PR.

Reply:
Hm I dislike the name too, although it's the name inside Windows environments. Let's do `vscode` for now and discuss using `code` in a new issue.

Reply:
@ayush9398, Please update it too as per asked by Makman2

Reply:
Oh I missed conv here, So `code` is the official alias/shortname of VSCode, I have seen on all distros I have tested that its the invocation command.

Reply:
Yeah but it's just `code`. That's kinda a very generic word, so I would refrain now from introducing it. Let's discuss on a separate issue.

Reply:
Still waiting for this to be changed to `vscode`, or explaination why that wont be good

Reply:
The default name for the application file which loads the application has name 'code', on changing 'code' to 'vscode', coala doesn't identify the application and hence, throws error.